wp-content/themes/citylimits/inc/block-color-palette.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Block color palette information
+ */
+/**
+ * Define the block color palette
+ *
+ * If updating these colors, please update less/vars.less. Slugs should match LESS var names.
+ *
+ * @link https://developer.wordpress.org/block-editor/developers/themes/theme-support/
+ * @return Array of Arrays
+ */
+function citylimits_block_colors() {
+	return array(
+		array(
+			'name' => __( 'White', 'citylimits' ),
+			'slug' => 'white',
+			'color' => 'white',
+		),
+		array(
+			'name' => __( 'Red', 'citylimits' ),
+			'slug' => 'red',
+			'color' => '#D41313',
+		),
+		array(
+			'name' => __( 'Dark Grey', 'citylimits' ),
+			'slug' => 'darkgrey',
+			'color' => '#333',
+		),
+		array(
+			'name' => __( 'Grey 1', 'citylimits' ),
+			'slug' => 'grey1',
+			'color' => '#c3c3c3',
+		),
+		array(
+			'name' => __( 'Grey 2', 'citylimits' ),
+			'slug' => 'grey2',
+			'color' => '#e2e2e2',
+		),
+		array(
+			'name' => __( 'Grey 3', 'citylimits' ),
+			'slug' => 'grey3',
+			'color' => '#666',
+		),
+		array(
+			'name' => __( 'Beige', 'citylimits' ),
```

Comment:
@benlk Regarding https://github.com/INN/umbrella-citylimits/pull/101/files#r344387643, `@beige` is right here

Reply:
I wonder what it says about me that I didn't see the color beige in the color list at all. It was just another shade of grey. It's even in half the screenshots I provided.